scripts/deploy_staging.ts

```diff
@@ -173,7 +176,22 @@ async function main() {
   await topUp.setUpkeepId(upkeepId);
   console.log("Upkeep ID set to DssVestTopUp contract");
 
-  console.log("TODO: Create a Uniswap pool for Test token and LINK");
+  const uniswapFactoryContract: Contract = new ethers.Contract(
+    UNISWAP_V3_FACTORY_ADDRESS,
+    abi
+  );
+
+  const createUniswapFactoryTx = await uniswapFactoryContract
+    .connect(admin)
```

Comment:
Isn't the first/admin account connected by default?

Reply:
I use ethers.Contract and not a contract factory so there's no signer connected by default and because of that it throws an error `'sending a transaction requires a signer` if I don't connect to this signer.